### PR TITLE
Value Tuner 2

### LINKED ISSUE #6: Add new valuetuner
Add the new custom valuetuner to the robot code that works through the networktables.

### PR DESCRIPTION
**Pull request by barel13**
Update the value constant to the latest version.
The new value tuner is off the robot code, and listens to the network tables and updates them accordingly

### CODE REVIEW

**Review by adamv17 — APPROVED** [review]
Very nice!!
Thank you (from Barel)

File: build.gradle [hunk 1]
```diff
@@ -79,10 +79,6 @@ dependencies {
     implementation name: 'jetty-all-9.4.0.v20161208-uber'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
 
-
-    implementation 'com.sparkjava:spark-core:2.8.0'
-    implementation 'com.sparkjava:spark-template-velocity:2.7.1'
-
```
Comment (katzuv) [inline build.gradle:85]: This change should be made in another branch.

File: src/main/java/frc/robot/valuetuner/NetworkTableConstant.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+/**
+ * The class is used to update the value of the web constant through the networktables.
+ */
+public class NetworkTableConstant implements WebConstant {
+    private static final NetworkTable BASE_TABLE = NetworkTableInstance.getDefault().getTable("value-tuner");
+
+    private final NetworkTableEntry constant;
+    private final double defaultValue;
+
+    NetworkTableConstant(String subsystem, String key, double defaultValue) {
```
Comment (katzuv) [inline src/main/java/frc/robot/valuetuner/NetworkTableConstant.java]: "subsystem" is too specific, you might want to have subtables for things that aren't subsystems.

**Review by katzuv — CHANGES_REQUESTED** [review]
Please explain the usage, what does "wrapper" mean? Why is `WrapperConstant` used in competition?

File: build.gradle (hunk 1 above)
Comment (barel13) [reply build.gradle:85]: No it's not, the dependencies only being used by the value tuner (and now they are not...)

**Comment by barel13** [comment]
> Please explain the usage, what does "wrapper" mean? Why is `WrapperConstant` used in competition?

Because we don't want to use the networktables during the competition

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 
+import frc.robot.valuetuner.WebConstant;
+
 public final class Constants {
 
     public static class ExampleSubsystem {
-        public static final double POWER = 0.5; // [%]
+        private static final String NAME = ExampleSubsystem.class.getName();
+        public static final WebConstant POWER = WebConstant.of(NAME, "power", 0.5); // [%]
     }
```
Comment (GaiaZano05) [inline src/main/java/frc/robot/Constants.java:20]: Delete empty line.

**Review by Se-gol — APPROVED** [review]

**Review by GaiaZano05 — APPROVED** [review]
You can do better.

**Pull request merged**